model/uzivatel.php

```diff
@@ -261,6 +261,49 @@ function jmeno()
     return $this->u['jmeno_uzivatele'].' '.$this->u['prijmeni_uzivatele'];
   }
 
+  /** Jméno uživatele */
```

Comment:
style: doc comment dává smysl jen tehdy, pokud má přidanou informaci. U getterů, kde víceméně jenom kopíruje název metody naopak jen prodlouží / zduplikuje text, takže se nehodí ho použít (pokud getter nedělá něco nezřejmého, co si zaslouží delší komentář). Proč dávat ho všude není dobrý nápad viz třeba https://blog.liquidbytes.net/2012/04/cargo-cult-coding-standards-cccs-for-php/